src/cgame/cg_rocket_draw.cpp

```diff
@@ -75,6 +75,65 @@ void Rocket_UpdateLanguage()
 		hudContext->Update();
 	updateLanguage = false;
 }
+
+// Standard HTML
+
+class LiElement : public Rml::Element
+{
+public:
+	LiElement(const Rml::String& tag) : Rml::Element(tag) {}
+
+private:
+	void OnUpdate() override
+	{
+		if ( !done )
+		{
+			SetInnerRML( va( "<li-bullet>• </li-bullet><li-content>%s</li-content>",
+				GetInnerRML().c_str() ) );
+			done = true;
+		}
+	}
+
+	bool done = false;
+};
+
+// Game-specific RML
+
+class WebElement : public Rml::Element
+{
+public:
+	WebElement(const Rml::String& tag) : Rml::Element(tag) {}
+
+private:
+	std::unordered_map<std::string, std::string> webUrls = {
+		{ "bugs", "bugs.unvanquished.net" },
+		{ "chat", "unvanquished.net/chat" },
+		{ "forums", "forums.unvanquished.net" },
+		{ "wiki", "wiki.unvanquished.net" },
+	};
+
+	void OnUpdate() override
+	{
+		bool found = false;
+
+		for ( const std::pair<const std::string, std::string>& p : webUrls )
+		{
+			if ( HasAttribute( p.first ) )
+			{
+				found = true;
+				SetInnerRML( va( "%s", p.second.c_str() ) );
```

Comment:
Useless invocation of `va`

Reply:
Ah yes, in my first version of the patch I was adding some `<b>` and stuff like that. 😁️

Reply:
Fixed.